cookbook/plans/enterprise.json

```diff
@@ -11,14 +11,14 @@
                     "denom": "ulava",
                     "amount": "600000000000"
                 },
-                "annual_discount_percentage": "0",
+                "annual_discount_percentage": 0,
                 "allow_overuse": false,
-                "overuse_rate": "0",
+                "overuse_rate": 0,
                 "plan_policy": {
-                    "geolocation_profile": "65535",

```

Comment:
needs to be "Global"

Reply:
done. see https://github.com/lavanet/lava/commit/2c666da79a87eeff3e849112b428e2549a828f8f